src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -46,6 +51,34 @@ export class Gu5xxPercentageAlarm extends GuAlarm {
       alarmDescription: props.alarmDescription ?? defaultDescription,
       evaluationPeriods: props.numberOfMinutesAboveThresholdBeforeAlarm ?? 1,
     };
-    super(scope, id, alarmProps);
+    super(scope, AppIdentity.suffixText(props, "High5xxPercentageAlarm"), alarmProps);
+  }
+}
+
+/**
+ * Creates an alarm which is triggered whenever there have been several healthcheck failures within a single hour.
+ */
+export class GuUnhealthyHostsAlarm extends GuAlarm {
+  constructor(scope: GuStack, props: GuTargetGroupAlarmProps) {
+    const alarmName = `Unhealthy instances for ${props.app} in ${scope.stage}`;
+    const alarmDescription = `${props.app}'s instances have failed healthchecks several times over the last hour.
```

Comment:
Possibly overkill, but should we mention suspending the `ReplaceUnhealthy` process on the ASG to allow you to connect to an instance w/out fear of it being taken away from under your feet?

Reply:
Nice idea! I wondered how much detail we should get into here; my advice at the moment is slightly vague. It almost feels worth linking out to a runbook or something, as this is a common problem that I've helped to debug across many apps/teams.